test/GADM.jl

```diff
@@ -109,3 +109,52 @@ end
         @test diff < 0.01
     end
 end
+
+function getmeshespolygon(geometry)
+    #converts [Float, Float] to Meshes Point object
+    topoint2f = x -> Meshes.Point(x)
+    coordinates = GeoInterface.coordinates(geometry)
+
+    if string(ArchGDAL.getgeomtype(geometry)) === "wkbMultiPolygon"
```

Comment:
Why do we need to check if it is a multi versus poly gon? By design the coordinates will always return a multi polygon, so no need to check this.